core/crypto/pedersen_hash_test.go

```diff
@@ -58,8 +58,7 @@ func TestPedersenArray(t *testing.T) {
 		// Contract address calculation. See the following links for how the
 		// calculation is carried out and the result referenced.
 		//
-		// https://docs.starknet.io/documentation/develop/Contracts/contract-address/
-		// https://alpha4.starknet.io/feeder_gateway/get_transaction?transactionHash=0x1b50380d45ebd70876518203f131a12428b2ac1a3a75f1a74241a4abdd614e8
```

Comment:
I couldn't find this transaction in https://voyager.online/

Reply:
This is because it was part of Goerli test network. After sepolia appearance Goerli is deprecated (see https://docs.starknet.io/starknet-versions/deprecated/ )